spec.bs

```diff
@@ -480,13 +484,13 @@ To <dfn>handle an issue response</dfn>, given [=request=] |request| and [=respon
 1. Let |header| be the result of [=header list/get|getting=] <a http-header>Sec-Private-State-Token</a> from |response|'s [=response/header list=].
 1. If |header| is empty, return null.
 1. [=header list/delete|Delete=] <a http-header>Sec-Private-State-Token</a> from |response|'s [=response/header list=].
-1. Let |unmasked tokens| be the result of carrying out the cryptographic procedures to obtain a list of unmasked tokens given |header|.
-1. If |unmasked tokens| is an error, return a [=network error=].
+1. Let |commitments| and |pretokens| be the variables used in [=append private state token issue request headers=]. 
```

Comment:
As mentioned in #207, please explicitly define these.

Reply:
Pretokens is a little hard since this can't be recalculated here and needs to be the one corresponding to the append pst issue request header. Is there a canonical way to pass that information from the append to the handle response? A hacky method would be to create a new user agent database to store the pretokens at append time and then look it up here.

Reply:
How is this done in Chromium?

Reply:
We have a issuance class in the network stack that lives between the client sending a request and getting a response that stores all the information internally.

Reply:
So, my first thought would be to stash this on the request, for example like it's done here: https://wicg.github.io/webpackage/loading.html#mp-http-network-or-cache-fetch

We can probably start with that.

Reply:
Added to request.